### PR TITLE
Move Addresses,Read{Maps,Modules} to new ModuleUtils and rename

### PR DESCRIPTION
**Pull request by dpallotti**
Following offline discussion after
https://github.com/google/orbit/pull/3945#discussion_r923436850. These files
didn't really fit in `ObjectUtils` and a new module seemed appropriate.

Bug: http://b/239512651

Test: Build.

### CODE REVIEW

File: src/CodeReport/Disassembler.cpp [hunk 1]
```diff
@@ -13,6 +12,7 @@
 #include <algorithm>
 
 #include "ClientData/ModuleAndFunctionLookup.h"
+#include "ModuleUtils/VirtualAndAbsoluteAddresses.h"
```
Comment (antonrohr) [inline src/CodeReport/Disassembler.cpp:15]: I guess this is not actually used in the file

**Comment by dpallotti** [comment]
I have some fixing to do it seems...

File: src/FakeClient/FakeClientMain.cpp [hunk 2]
```diff
@@ -29,8 +29,8 @@
 #include "GraphicsCaptureEventProcessor.h"
 #include "GrpcProtos/Constants.h"
 #include "GrpcProtos/capture.pb.h"
+#include "ModuleUtils/ReadLinuxModules.h"
```
Comment (antonrohr) [inline src/FakeClient/FakeClientMain.cpp:32]: Same here. ModuleUtils into FakeClient cmake

**Comment by dpallotti** [comment]
Some `SymbolAbsoluteAddressToVirtualAddress` were turned into `SymbolVirtualAddressToAbsoluteAddress`. Fixing...

**Comment by dpallotti** [comment]
> Some `SymbolAbsoluteAddressToVirtualAddress` were turned into `SymbolVirtualAddressToAbsoluteAddress`. Fixing...

That was awkward. Should be fixed now.

File: src/CodeReport/DisassemblerTest.cpp [hunk 3]
```diff
@@ -2,11 +2,11 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#include <ObjectUtils/Address.h>
 #include <gtest/gtest.h>
 
 #include "AssemblyTestLiterals.h"
 #include "CodeReport/Disassembler.h"
+#include "ModuleUtils/VirtualAndAbsoluteAddresses.h"
```
[the remark this answers is not in the document]
Comment (dpallotti) [reply src/CodeReport/DisassemblerTest.cpp:9]: This dependency is only for the tests actually, I only added it to the corresponding target.

**Review by beckerhe — APPROVED** [review]
LGTM

**Review by florian-kuebler — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**